### PR TITLE
Added a "Delete all answers" button to frontpage

### PR DESCRIPTION
**Pull request by dattvidc**
And the button deletes all answers from localstorage 

Method: localstorage.clear()

### CODE REVIEW

**Comment by dattvidc** [comment]
Obviously was not aware you would be able to see my git saves comments.....sorry

**Comment by dattvidc** [comment]
Is there anything else I need to do ? ?

File: src/utils/persistAnswers.js [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
 };
 
 export const clearPersistedAnswer = title => {
-  typeof window !== 'undefined' &&
-    localStorage.removeItem(title);
+    typeof window !== 'undefined' &&
+        localStorage.removeItem(title);
 };
+
+export const clearAllPersistedAnswer = title => {
```
Comment (nas5w) [inline src/utils/persistAnswers.js]: This doesn't need to take a title argument, right?

File: src/utils/persistAnswers.js [hunk 2]
```diff
@@ -1,26 +1,31 @@
 export const persistAnswer = (
-  title,
```
Comment (nas5w) [inline src/utils/persistAnswers.js]: There's quite a bit of changes here due to your local environment adding extra indentation. Can you undo that?

File: src/utils/persistAnswers.js (hunk 1 above)
Comment (dattvidc) [reply src/utils/persistAnswers.js]: (making facepalm) no...no it doesnt...

File: src/utils/persistAnswers.js (hunk 2 above)
Comment (dattvidc) [reply src/utils/persistAnswers.js]: No..apparently not. I will copy the original file and redo my changes..sorry...

**Comment by dattvidc** [comment]
Will make new

**Pull request closed without merging**